README.md

```diff
@@ -1,141 +1,133 @@
+```
+# with npm
+cd playground
+npm install
+npm run develop
+```
 
-No additional routes should be created on strapi for the front end users as they will be using the MeiliSearch API to search and not Strapi (unless we want to make this possible?).
+Install Strapi with this **Quickstart** command to create a Strapi project instantly:
 
-## MVP
 
-### MVP of MVP
-- Downloadable plugin
-- Communication with a running MeiliSearch
-- Interface with the following: 
-    - Add host and keys
-    - Ability to chose which collections should be indexed in MeiliSearch
-    - Every time a collection is updated, the updated element should be updated in MeiliSearch as well (listener to add, edit and delete)
-    - Possibility to remove a collection from MeiliSearch
+#### Create strapi project
 
-### MVP ++ 
+- (Use **yarn** to install the Strapi project (recommended). [Install yarn with these docs](https://yarnpkg.com/lang/en/docs/install/).)
 
-Same as Above with two small differences: 
-- Add settings (json file) 
-- Settings are added through a nicer interface
-- Possibility to add relations between Collections. Exemple: Given 2 collections, `restaurants` and `users`, facet
+```bash
+# with yarn
+yarn create strapi-app my-project --quickstart
 
-### Bonus 
+# with npm/npx
+npx create-strapi-app my-project --quickstart
+```
 
-- Possibility for the user to click `add to meilisearch` on Collection creation
+_This command generates a brand new project with the default features (authentication, permissions, content management, content type builder & file upload). The **Quickstart** command installs Strapi using a **SQLite** database which is used for prototyping in development._
 
-## STEPS PAR PHASE
 
-### Architecture 
-- Playground -> app strapi
-- src -> contain plugin
-- tests
+Once your strapi project has been created, to link the plugin to this project you have to create a symbolic link inside a plugin folder at the root of the strapi project.
 
-### PRE A
-- [ ] Petit README 
-- [ ] Linter
-- [ ] Contributing
-- [ ] How do you add plugin to Strapi
+1. Create plugin folder
+```
+mkdir plugins
+```
+2. Create symbolic link
+```
+cd plugins
+ln -s [PATH_TO_PLUGIN] meilisearch
+```
+3. Develop
+```
+yarn develop
+```
 
-### A
+You can now use the plugin on your strapi project.
 
-- [ ] Create local plugin
-- [ ] Create back-end communication with MeiliSearch (Connection, index creation, doc addition)
-- [ ] Create UI for HOST and KEY on dashboard
-- [ ] Create UI for index name on dashboard
-- [ ] Create UI for document addition (in STEP A you have to provide a json string)
+## 🖐 Requirements
 
--> This Creates a working version of MeiliSearch on Strapi but does not communicate with strapi yet
+Complete installation requirements are exact same as for Strapi itself and can be found in the documentation under <a href="https://strapi.io/documentation/v3.x/installation/cli.html#step-1-make-sure-requirements-are-met">Installation Requirements</a>.
```

Comment:
Why don't you use the markdown link? This is easier to read (for maintainer)

Reply:
It was a copy paste from another readme, changing that

Reply:
It was a copy paste from another strapi readme ! Changed it